woocommerce/woocommerce
# [e2e tests] README.md - Add link to WooCommerce Monorepo prerequisites

Add a link to the WooCommerce Monorepo prerequisites before going through the e2e-pw prerequisites.

### Changes proposed in this Pull Request:

Following the WooCommerce E2E environment set up steps could lead to an error by not having some of the dependencies required to build the WooCommerce project.

Specifically, Composer is not included among the prerequisites to set up WooCommerce E2E Playwright environment, which likewise requires PHP. For that reason, installing the dependencies led to an error due to not not having Composer installed. Going to the steps described in the WooCommerce Monorepo prerequisites is needed beforehand.

This PR aims to include a link to those [WooCommerce Monorepo prerequisites](https://github.com/woocommerce/woocommerce/blob/trunk/README.md#prerequisites), while avoiding duplicated content by using documentation which is already available.

For additional context, please refer to [this Slack thread](https://a8c.slack.com/archives/C01BWDDTGKX/p1704893298832459).

### How to test the changes in this Pull Request:

The only way to reproduce the issue 

Run a clean installation to set up WooCommerce Playwright End to End Tests following the updated documentation.

1. PRE: WooCommerce project and its dependencies are not available
2. Follow [WooCommerce Playwright End to End Tests introduction](https://github.com/woocommerce/woocommerce/tree/trunk/plugins/woocommerce/tests/e2e-pw#introduction)
3. Notice the results running `pnpm install`, which should install required dependencies
4. No error message should show up

### Changelog entry




<details>

#### Significance


#### Type


#### Message


#### Comment 

Update E2E Playwright environment README.md file with a link to the WooCommerce Monorepo prerequisites to avoid duplicate content.

</details>
